src/maliput/routing/route.cc

```diff
@@ -360,5 +365,130 @@ int Route::FindDirectionTowardsLaneSRangeWithStraightPredecessor(const Route::La
   return kTowardsRight;
 }
 
+namespace {
+
+// Finds a RoadPosition by matching @p lane pointer.
+// @param lane Pointer to the api::Lane to match.
+// @param positions Vector or positions to look for one with a matching api::Lane pointer with @p lane.
```

Comment:
"or positions" should be "of positions".

Reply:
[5f99c1c](https://github.com/maliput/maliput/pull/628/commits/5f99c1c613ff4ae3206bf0796b0287698f0d08ae)